worlds/sm64ex/Rules.py

```diff
@@ -246,6 +247,7 @@ class RuleFactory:
 
     token_table = {
         "TJ": "Triple Jump",
+        "DJ": "Triple Jump",
```

Comment:
Is this supposed to be Triple Jump?

Reply:
"Triple Jump" the item unlocks both "DJ (Double Jump)" and "TJ (Triple Jump)" in the logic.  Doesn't serve a purpose in the rules currently, but it lets anyone looking at the file know that DJ requires Triple Jump.

This was going to be part of a greater logic rework that replaced TJ with DJ in some rules to future-proof the logic for when DJ gets the option to be randomized separately, but that's a big feature addition